q2_types/feature_data/_transformer.py

```diff
@@ -168,6 +168,7 @@ def _6(ff: TaxonomyFormat) -> pd.Series:
 @plugin.register_transformer
 def _28(ff: TaxonomyFormat) -> qiime2.Metadata:
     df = _taxonomy_formats_to_dataframe(str(ff), has_header=None)
+    df['Taxon'] = df['Taxon'].str.strip()
```

Comment:
Should this step happen somewhere in `_taxonomy_formats_to_dataframe`, instead?

Reply:
I think you are right. It would make sense to perform this step before using a transformer. I will submit a new commit, with the strip function, right before the return statement.